my-sms/app/controllers/students_controller.rb

```diff
@@ -1,8 +1,10 @@
+# frozen_string_literal: false
+
```

Comment:
This is the default option (you would use `frozen_string_literal: true` to override), unless you're setting `RUBYOPT=--enable-frozen-string-literal`?

Reply:
I have some rubocop autoformatting and this was cleaner than disabling the cop.

Reply:
Ah, all good 👍